### ISSUE TITLE
docker: gradle.properties file

### ISSUE THREAD

**Issue by rdmueller** [docToolchain/docToolchain#684]
it would be nice if docker would recognize the `gradle.properties` file of the project.

To implement this, we have to copy the `/home/dtcuser/project/gradle.properties` file to `$GRADLE_HOME`

To ensure that this is not shared between projects, the `gradle.properties` should be removed before and after each run

**Comment by jmini** [comment]
Using a `gradle.properties` defined in the project also do not work in with the local mode. Right?

**Comment by rdmueller** [comment]
yes and no. as far as I know, you can use a `gradle.properties` file in your home folder and thus set global creadentials for gradle.
